packages/manager/modules/overthebox/src/overthebox/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "overTheBox_page_title": "Mon OverTheBox \"{{name}}\"",
+  "overTheBox_title": "OverTheBox",
+  "overTheBox_back": "Retour",
+  "overTheBox_tabs_detail": "Mon OverTheBox",
+  "overTheBox_tabs_remote": "Accès à distance",
+  "overTheBox_tabs_tasks": "Tâches",
+  "overTheBox_tabs_docs": "Guides",
+  "overTheBox_tabs_logs": "Logs",
+  "overTheBox_no_device_found": "Aucune OverTheBox n'est branchée. Vous ne pourrez pas créer d'accès à distance, ni utiliser les accès créés. Pour vous aider lors de la configuration de votre OverTheBox, veuillez trouver le guide d'installation <a href=\"{{url}}\" target=\"_blank\" title=\"Nouvelle fenêtre\">ici</a>."
```

Comment:
You say here : `"Aucune OverTheBox"`(Feminine), but above it's written `"Mon OverTheBox"`(Masculine) ?

Reply:
I've only moved this translations

Reply:
Ok